tests/etl/google_sheet_to_athena.py

```diff
@@ -10,7 +10,7 @@
 class TestS3Util(TestCase):
     @classmethod
     def setUpClass(cls):
-        cls.util = GoogleSheetToAthena(GoogleSheetsToAthenaSettings(
+        cls.settings = GoogleSheetsToAthenaSettings(
             keys_object=None,
             workbook_name='Tradie Acquisition Targets',
```

Comment:
should this be a test workbook to avoid collisions with real world workbooks

Reply:
fixed